src/mono/browser/build/BrowserWasmApp.targets

```diff
@@ -358,7 +358,8 @@
       <EmscriptenEnvVars Include="PYTHONPATH=$(EmscriptenPythonToolsPath)" Condition="'$(OS)' == 'Windows_NT'" />
       <EmscriptenEnvVars Include="PYTHONHOME=" Condition="'$(OS)' == 'Windows_NT'" />
       <EmscriptenEnvVars Include="EM_CACHE=$(WasmCachePath)" Condition="'$(WasmCachePath)' != ''" />
-      <EmscriptenEnvVars Include="EM_FROZEN_CACHE=True" Condition="'$(WasmCachePath)' == '$(EmscriptenCacheSdkCacheDir)'" />
+      <EmscriptenEnvVars Include="EM_FROZEN_CACHE=1" Condition="'$(WasmCachePath)' == '$(EmscriptenCacheSdkCacheDir)'" />
```

Comment:
In order to make it work also for runtime build we also need it in `src\mono\browser\browser.proj`

Together with other `<_CmakeEnvironmentVariable`

Reply:
I will address it in a follow up PR

Reply:
https://github.com/dotnet/runtime/pull/106188